queue_services/business-bn/pyproject.toml

```diff
@@ -0,0 +1,145 @@
+[tool.poetry]
+name = "business-bn"
+version = "0.1.0"
+description = ""
+authors = ["Patrick Wang <peinan.wang@gov.bc.ca>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
```

Comment:
For next time (don't need to change it for this PR): you have a lot of extra things defined in here that poetry can resolve for us automatically. I usually start with the big ones like flask and then add in the few dependencies needed afterwards (like psycopg2). You end up with a much smaller list here that's easier to manage going forward. Also if you make use of the 'poetry init' and 'poetry add' commands it'll format this differently in the new version